string title
uint8 typeXoY   # 0 für x, 1 für y
float32 maxT
float32 dT
float32 k
float32[2] xrange
float32[2] yrange
---